requirements/constraints.txt

```diff
@@ -20,4 +20,7 @@ twine<2.0.0
 zipp<2.0.0
 
 # Requires: Python >=3.6
-inflect<4.0.0
\ No newline at end of file
+inflect<4.0.0
+
+# Requires: Python >=3.8
+tox-battery==0.5.2
```

Comment:
Instead of pinning it to just 0.5.2. It's easier to read to do `tox-battery<0.6.0